lib/ioki/model/platform/task.rb

```diff
@@ -15,9 +15,9 @@ class Task < Base
         attribute :completer, on: :read, type: :object
         attribute :completer_id, on: :read, type: :string
         attribute :completer_type, on: :read, type: :string
-        attribute :pause, on: :read, type: :object, class_name: 'Pause'
+        attribute :pause_id, on: :read, type: :string
         attribute :rejectable, on: :read, type: :boolean
-        attribute :ride, on: :read, type: :object, class_name: 'Ride'
+        attribute :ride_id, on: :read, type: :string
```

Comment:
Is this no longer nested? Because with other cases it usually is.

Reply:
Same as above :D